api/accounts.go

```diff
@@ -0,0 +1,137 @@
+package api
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	recurly "github.com/blacklightcms/go-recurly"
+)
+
+var _ recurly.AccountsService = &AccountsService{}
+
+// AccountsService handles communication with the accounts related methods
+// of the recurly API.
+type AccountsService struct {
+	client *Client
+}
+
+const (
```

Comment:
Fixed in [56f7a0](https://github.com/blacklightcms/go-recurly/pull/10/commits/70680f140c93eac4f28ac08599257fe42756f7a0)